tests/ContentInterpolationTest.php

```diff
@@ -297,6 +297,9 @@ public function testPHPBlock54()
     {
         if (version_compare(PHP_VERSION, '5.4.0') < 0) $this->markTestSkipped('PHP < 5.4');
 
+        ini_set('short_open_tag', 0);
+        if (ini_get('short_open_tag')) $this->markTestSkipped("PHP is buggy");
```

Comment:
Honestly I can't get what `testPHPBlock54` should test, but this seems now needed to get it pass

Reply:
@Potherca do you perhaps know what's going on here?